react-notes/src/components/Notepad/notepadContainer.jsx

```diff
@@ -1,19 +1,56 @@
 import PropTypes from 'prop-types';
-import { useEffect, useMemo, useState } from 'react';
-import MockData from '../../mock/mockNotes';
+import { useEffect, useState } from 'react';
+
+import { EMPTY_NOTE } from '../../config/constants';
+import { makeDate } from '../../utils/utils';
 import Notepad from './Notepad';
 
-const NotepadContainer = ({ noteId }) => {
-  const mockData = useMemo(() => new MockData(), []);
+const NotepadContainer = ({ noteId, onUpdate, onDelete }) => {
   const [currNote, setCurrNote] = useState();
+  const [isEdit, setEdit] = useState(false);
+  const [isSave, setSave] = useState(false);
+  const [noteText, setText] = useState(EMPTY_NOTE);
+  const [noteTitle, setTitle] = useState(EMPTY_NOTE);
+  useEffect(() => {
+    if (isSave) {
+      const updatedDate = makeDate();
+      const updatedNote = {
+        title: `${noteTitle || currNote.title}`,
+        content: `${noteText || currNote.content}`,

```

Comment:
Do we really need to use template here?
Maybe we can write it just using: 
```suggestion
        title: noteTitle || currNote.title,
        content: noteText || currNote.content,

```